File: Company-gRPC-service/src/main/java/grpc/project/companygrpcservice/Project/Project.java
package grpc.project.companygrpcservice.Project;

import grpc.project.companygrpcservice.Employee.Employee;
import grpc.project.companygrpcservice.Enums.StatusProject;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Collection;
import java.util.Date;
@Data @AllArgsConstructor @NoArgsConstructor @Builder
public class Project {
    private String Id;
    private String name;
    private Date deadLine;
    private boolean finished;
    private Collection<Employee> employees;
    private StatusProject statusProject;
}

File: Employee-gRPC-service/Models/Project/Project.cs
using Employee_gRPC_service.Enums;

namespace Employee_gRPC_service.Models {
    public class Project {
        public string? Id{get; set;}
        private string? name;
        private DateTime? deadLine;
        private Boolean finished;
        private ICollection<Employee>? employees;
        private StatusProject statusProject;
    }
}